Jorge/HTML & CSS/styles.css

```diff
@@ -0,0 +1,142 @@
+body {
+  background-color: lightblue;
+  display: flex;
+  justify-content: center;
+}
+.main-container {
+  background-color: grey;
+  width: 80%;
+}
+.header {
+  display: flex;
+  background-color: white;
+  margin: 1rem;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+.header__logo {
+  flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.header__logo img {
+  max-width: 50%;
+  height: auto;
+}
+.header__menu {
+  flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.header__menu ul{
+  list-style-type: none;
+  display: flex;
+  background-color: grey;
+  justify-content: center;
+  padding-left: 0;
+}
+.header__menu ul > li{
+  padding: 1rem;
+}
+.header__menu ul > li a {
+  text-decoration: none;
+}
+.header__button {
+  flex: 2;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+}
+.header__button button:first-child {
+  margin-right: 1rem;
+}
+.button {
+  padding: 0.5rem;
+  background-color: #ffffff;
+  cursor: pointer;
+}
+.primary {
+  background-color: blue;
+  color: #ffffff;
+}
+.hero {
+  border: black 1px solid;
+  background: url(images/wasteland.png);
+  padding: 2rem;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
```

Comment:
wouldn't `background-size: 100%` work the same?

Reply:
Apparently not. I tested visually and just with one 100% the images fits the whole size of its container but it is not possible to see the entire image. With 100% 100% the image it is adjusted horizontally and vertically the whole size of the container and with the correct proportions as the original image.